pulser-core/pulser/backend/remote.py

```diff
@@ -81,14 +99,14 @@ def __init__(
         job_ids: list[str] | None = None,
     ):
         """Instantiates a new collection of remote results."""
-        self._submission_id = submission_id
+        self._batch_id = submission_id
         self._connection = connection
         if job_ids is not None and not set(job_ids).issubset(
-            all_job_ids := self._connection._get_job_ids(self._submission_id)
+            all_job_ids := self._connection._get_job_ids(self._batch_id)
         ):
             unknown_ids = [id_ for id_ in job_ids if id_ not in all_job_ids]
             raise RuntimeError(
-                f"Submission {self._submission_id!r} does not contain jobs "
+                f"Submission {self._batch_id!r} does not contain jobs "
```

Comment:
The error should mention batch instead of submission, no ?

Reply:
Good question, I didn't change it to avoid mixing up "batch" and "submission" within the same class, but I guess here it wouldn't be so bad